packages/network-controller/tests/provider-api-tests/no-block-param.ts

```diff
@@ -921,7 +961,9 @@ export function testsForRpcMethodAssumingNoBlockParam(
           },
         );
 
-        expect(result).toBeUndefined();
+        await expect(promiseForResult).rejects.toThrow(
```

Comment:
Should the test description here be updated to say "throws" instead of "returns"? Likewise for the other test updates in this module that now throw when they hadn't previously.

Reply:
Hmm, interesting. I guess it depends on how people are likely to think about methods that return promises. When used with `await`, they act like synchronous methods, so we could say that they "return X" when they resolve and they "throw Y" when they reject. Or we could say that they "return a promise that resolves with X" or "return a promise that rejects with Y". Do you think it's worth it to be accurate or does it not matter?

Reply:
Hmm, I tend to prefer the shorter form just for brevity but I don't have a strong preference either way. Either way would be a lot more clear than the current description